packages/ember-test-app/app/templates/faq.hbs

```diff
@@ -0,0 +1,15 @@
+{{page-title "FAQs"}}
+<main data-theme="marketing" class="bg-primary-subtle vh-100 vw-100">
+  <div class="container p-5">
+    <Mktg::Faq
+      @open={{true}}
```

Comment:
Can we change this variable name to something like `defaultOpen`?
I want to make it clear that this variable only controls the initial state of the component.